bank2ynab/dataframe_handler.py

```diff
@@ -123,18 +123,20 @@ def read_csv(
     :return: Dataframe read from CSV file
     :rtype: pd.DataFrame
     """
-    df = pd.read_csv(
-        file_path,
-        delimiter=delim,
-        skipinitialspace=True,  # skip space after delimiter
-        names=[],  # don't set column headers initially
-        skiprows=header_rows,  # skip header rows
-        skipfooter=footer_rows,  # skip footer rows
-        skip_blank_lines=True,  # skip blank lines
-        encoding=encod,
-        memory_map=True,  # access file object directly - no I/O overhead
-        engine="python",
-    )
+
+    with open(file_path, 'rU') as csv_file:
```

Comment:
I was experiencing an issue with pandas.read_csv throwing `new-line character seen in unquoted field - do you need to open the file in universal-newline mode?` seemingly because my bank was formatting files using unix style line endings, instead of windows. Adding the `rU` universal line endings flag and using a file handle resolves this.
`